src/modules/billing/services/batch-service.js

```diff
@@ -168,7 +168,7 @@ const approveBatch = async (batch, internalCallingUser) => {
     await licencesService.updateIncludeInSupplementaryBillingStatusForSentBatch(batch.id);
     await invoiceService.resetIsFlaggedForRebilling(batch.id);
 
-    return setStatus(batch.id, BATCH_STATUS.processing);
+    return batch;
   } catch (err) {
     logger.error('Failed to approve the batch', err, batch);
```

Comment:
Do you think we need to update the batch status in this `catch()` block?

I'm wondering if we might need to set it to `error`.

Reply:
I think we touched on this point when we discussed the proposed solution and I believe we agreed to not set it to error so that the user can retry to send/approve the batch if the process failed? If we set the batch to error they would have to re-run the batch. I am not sure - but happy to add it if you think we should

Reply:
However the batch status will be 'processing' when it reaches this code. so if the process fails here we might need to set it back to ready?